packages/mwp-api-proxy-plugin/src/routes.js

```diff
@@ -26,6 +27,18 @@ const getApiProxyRoutes = path => {
 				'electrode-csrf-jwt': {
 					enabled: true,
 				},
+				'mwp-tracking-plugin': {
+					click: request => {
+						// only consume click data when the querystring indicates that it should be consumed
+						// this allows the client to avoid invoking click parsing for lazy-loaded API requests
```

Comment:
This is a nitpick but I found this comment a bit confusing. I'd probably just remove that second line.